selfdrive/ui/ui.cc

```diff
@@ -181,31 +181,31 @@ static void update_sockets(UIState *s) {
     scene.gpsOK = sm["liveLocationKalman"].getLiveLocationKalman().getGpsOK();
   }
   if (sm.updated("carParams")) {
-    s->longitudinal_control = sm["carParams"].getCarParams().getOpenpilotLongitudinalControl();
+    scene.longitudinal_control = sm["carParams"].getCarParams().getOpenpilotLongitudinalControl();
   }
   if (sm.updated("driverState")) {
     scene.driver_state = sm["driverState"].getDriverState();
   }
   if (sm.updated("driverMonitoringState")) {
     scene.dmonitoring_state = sm["driverMonitoringState"].getDriverMonitoringState();
-    if(!scene.driver_view && !s->ignition) {
+    if(!scene.driver_view && !scene.ignition) {
       read_param(&scene.driver_view, "IsDriverViewEnabled");
     }
-  } else if ((sm.frame - sm.rcv_frame("driverMonitoringState")) > UI_FREQ/2) {
+  } else if (scene.driver_view && (sm.frame - sm.rcv_frame("driverMonitoringState")) > UI_FREQ/2) {
```

Comment:
Why did this change?

Reply:
The previous version had this. after this PR was submitted, the master has changed too much